src/main/app/pdf/mappers/claimMapper.ts

```diff
@@ -11,7 +11,7 @@ export class ClaimMapper {
       claimNumber: claim.claimNumber,
       amount: NumberFormatter.formatMoney(claim.claimData.amount.totalAmount()),
       issueFee: NumberFormatter.formatMoney(claim.claimData.paidFeeAmount),
-      totalAmount: NumberFormatter.formatMoney(claim.totalAmount),
+      totalAmount: NumberFormatter.formatMoney(claim.totalAmountTillToday),
```

Comment:
Are you sure this should be changed as well? The change was only to update part admission pages and first contact view of the claim.